tests/drivers/counter/counter_basic_api/src/test_counter.c

```diff
@@ -188,42 +198,45 @@ void test_single_shot_alarm_instance(const char *dev_name, bool set_top)
 	}
 
 	err = counter_start(dev);
-	zassert_equal(0, err, "Counter failed to start\n");
+	zassert_equal(0, err, "%s: Counter failed to start\n", dev_name);
 
 	if (set_top) {
 		err = counter_set_top_value(dev, ticks, top_handler,
 					    exp_user_data);
 
-		zassert_equal(0, err, "Counter failed to set top value\n");
+		zassert_equal(0, err,
+			     "%s: Counter failed to set top value\n", dev_name);
 
 		alarm_cfg.ticks = ticks + 1;
 		err = counter_set_channel_alarm(dev, 0, &alarm_cfg);
 		zassert_equal(-EINVAL, err,
-			      "Counter should return error because ticks"
-			      " exceeded the limit set alarm\n");
-		alarm_cfg.ticks = ticks - 1;
```

Comment:
Was the removal of this line intentional? It is not mentioned in the commit message.

Reply:
no, i was cherry-picking it from other branch, probably automerge did that.

Reply:
fixed